llvm/utils/gn/secondary/llvm/lib/Transforms/IPO/BUILD.gn

```diff
@@ -33,6 +33,7 @@ static_library("IPO") {
     "DeadArgumentElimination.cpp",
     "ElimAvailExtern.cpp",
     "EmbedBitcodePass.cpp",
+    "ExpandVariadics.cpp",
```

Comment:
There's a bot that updates the gn files automatically... so unless you're building this way, I'd suggest not touching this file.

Reply:
Nice, I did not know what. I am indeed not a BUILD.gn user.